src/clj/puppetlabs/services/master/master_service.clj

```diff
@@ -17,6 +17,7 @@
    [:CaService initialize-master-ssl! retrieve-ca-cert! retrieve-ca-crl! get-auth-handler]
    [:JRubyPuppetService]
    [:AuthorizationService wrap-with-authorization-check]
+   [:SchedulerService interspaced stop-job]
```

Comment:
FYI, I don't think we need `stop-job`....so I'm going to remove it. I started up puppet-server in the repl, then used the `(stop)` function to confirm that the stop function is called as well for tk-scheduler and it looks like it is.

```
...
2016-07-20 11:41:18,300 DEBUG [async-dispatch-5] [p.t.s.s.scheduler-service] Shutting down Scheduler Service
2016-07-20 11:41:18,300 DEBUG [async-dispatch-5] [p.t.s.s.scheduler-service] Scheduler Service shutdown complete.
...
```